### PR TITLE
Updating xUnit Benchmark Scenarios to capture Etw with Pmc support.

### PR DESCRIPTION
**Pull request by jorive**
- Adding 'tiered' jitting
- Enable JitBench to capture Pmc data
- Fixed some xUnit performance warnings

### CODE REVIEW

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
@dotnet/rap-team PTAL

**Comment by adiaaida** [comment]
Looks pretty good to me.

File: tests/scripts/run-xunit-perf.cmd [hunk 1]
```diff
@@ -383,6 +378,11 @@ rem ****************************************************************************
 rem   Generates BenchView's submission data and upload it
 rem ****************************************************************************
 setlocal
+  REM if not exist measurement.json (
```
[the remark this answers is not in the document]
Comment (jorive) [reply tests/scripts/run-xunit-perf.cmd]: Good catch! I'll uncomment it :)

---
In reply to: [147792827](https://github.com/dotnet/coreclr/pull/14734#discussion_r147792827) [](ancestors = 147792827)

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
I am retesting and making sure nothing breaks (I had to resolve conflicts on `dependencies.props`)

**Comment by jorive** [comment]
@dotnet-bot test ci please

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT x64 full_opt perf
@dotnet-bot test Windows_NT x64 full_opt perf scenarios
@dotnet-bot test linux perf

**Comment by jorive** [comment]
@dotnet-bot help please

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT x64 perf scenarios

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT x64 perf scenarios

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT x64 perf scenarios

**Pull request merged**